tidb_binlog/binlogctl/README.md

```diff
@@ -41,39 +42,49 @@ cd tidb-tools-latest-linux-amd64
 
 ## Example
 
+### Query Pump/Drainer
 
+Run the following command:
 
-### query pumps/drainers
 ```
 bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd pumps/drainers
```

Comment:
I suppose this means "`-cmd pumps` or `-cmd drainers`", not literally "`-cmd pumps/drainers`" right? If former, could we clarify e.g. by writing them in separate commands?

```sh
bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd pumps
bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd drainers
```

Reply:
@GregoryIan ?

Reply:
yep